### DISCUSSION TITLE
Track Platform idea dump

### DISCUSSION THREAD

**Discussion by slambo311** [BEEmod/BEE2-items#4118] [Feature Suggestions]
1. Alternate Track Platform that's just an elevator, with "Start Reversed" opening/closing the elevator doors.
2. Turns on the track, similar to turns on vac tubes.
2a. It might result in multiple platforms on one track, which could be fixed with a timer to choose an "ID" for the platforms (meaning platforms will not go onto a track that doesn't match its ID), and a toggle to remove platforms.
2b. A toggle to enable/disable the platform turning with the track could be used with this.
2c. To make it (probably) easier to code, it should be a very sharp turn. This graph shows what I mean. The one on the right is a sharp turn, while the left is a (crappily made) round turn.
![image](https://user-images.githubusercontent.com/61334045/164729766-01203cc2-2aa5-4f8b-942a-c1d20c311492.png)

**Comment by SimpleFoxOfficial** [comment]
and elevator will supports the vac tubes that you can wire to elevator

**Reply by slambo311** [reply]
That would probably be pretty hard to code, and elevators would likely clip through the vac tubes when turning. But that does give me another idea. Making vac tubes that are placed like tracks.

**Reply by SimpleFoxOfficial** [reply]
hmmm? like tracks? I don't understand but sounds good.

**Reply by slambo311** [reply]
Normally, you have to manually place vac tubes on each block and connect the vac tubes to each other. Tracks have a little slider to make them longer, and they're automatically connected.

**Reply by SimpleFoxOfficial** [reply]
hmm, yes, track slider is cool, but I didn't seen any test elements in BEE2.4, that have it. I think this is so hard to realize
